codebasin/util.py

```diff
@@ -141,6 +113,90 @@ def valid_path(path):
     return valid
 
 
+def _validate_json(json_object: str, schema_name: str) -> bool:
+    """
+    Validate JSON against a schema.
+
+    Parameters
+    ----------
+    json_object : Object
```

Comment:
Docstring hint (`Object`) doesn't match the function type hint (`str`). Also, usage in `config.py` passes a `dict` I think?

Reply:
Fixed in 1842cb969bb41dddf8311294583bd641f6eac604.

You're right that we tend to pass in a `dict`, but some of the usages allow other objects.  For example, a compilation database is an `array`.